feelslikemonday/app/src/main/java/com/example/feelslikemonday/ui/login/LoginMainActivity.java

```diff
@@ -18,6 +25,15 @@ protected void onCreate(Bundle savedInstanceState) {
     }
 
     public void attemptLogin(View view){
+        loginUsername = findViewById(R.id.editText);
+        pref = getApplicationContext().getSharedPreferences(SignupActivity.PREFS_NAME, 0);
+        SharedPreferences.Editor editor = pref.edit();
+        editor.putString(SignupActivity.USERNAME_KEY, loginUsername.getText().toString());
+        userNameShared = loginUsername.getText().toString();
+        editor.commit();
+
+        String temp;
+        temp = pref.getString(SignupActivity.USERNAME_KEY,null);

```

Comment:
I don't think temp is used anywhere else. Is this outdated?

Reply:
yes it was for something I was testing and i forgot it there